src/mlpack/methods/cf/svd_wrapper_impl.hpp

```diff
@@ -9,8 +9,20 @@
  * 3-clause BSD license along with mlpack.  If not, see
  * http://www.opensource.org/licenses/BSD-3-Clause for more information.
  */
+
+#ifndef MLPACK_METHODS_SVDWRAPPER_IMPL_HPP
+#define MLPACK_METHODS_SVDWRAPPER_IMPL_HPP
+
+// In case it hasn't yet been included.
+#include "svd_wrapper.hpp"
+
+namespace mlpack
+{
+namespace cf
+{
+
 template<class Factorizer>
-double mlpack::cf::SVDWrapper<Factorizer>::Apply(const arma::mat& V,
+double SVDWrapper<Factorizer>::Apply(const arma::mat& V,
```

Comment:
Not a doxygen issue, but maybe you can align the parameters with the first one.

Reply:
Thanks for noticing, fixed with 799b7e2